lib/helpers.js

```diff
@@ -115,6 +116,14 @@ export function firstImplementer(...targets) {
   });
 }
 
+function isRoot(dir) {
+  return path.resolve(dir, '..') === dir;
+}
+
+export function isValidWorkdir(dir) {
+  return dir !== os.homedir() && !isRoot(dir);
```

Comment:
Note that this won't handle things like relative paths or symlinks.

That's probably okay for this, though, since we just want to explicitly exclude `~` and `/` mistakes, and if someone slips one of them through with filesystem trickery the TooLarge state will catch it.